WcaOnRails/app/models/user.rb

```diff
@@ -468,6 +468,10 @@ def can_confirm_competition?(competition)
     can_admin_results? || competition.delegates.include?(self)
   end
 
+  def can_submit_competition_results?(competition)
+    competition.delegates.include?(self)
```

Comment:
If you look at all our other `can_*` methods, we always give admins the power to do stuff, so I'd rewrite this as:

```ruby
can_admin_results? || competition.delegates.include?(self)
```

Reply:
Good point; I've changed it